core/src/main/java/org/elasticsearch/ingest/core/Pipeline.java

```diff
@@ -77,6 +80,11 @@ public String getDescription() {
         return compoundProcessor.getOnFailureProcessors();
     }
 
+    public Map<String, Object> toMap() {
+        Map<String, Object> result = new HashMap<>();
+        return result;
```

Comment:
leftover from trying to make pipeline/processors know how serialise themselves :)